bb-aem-integration-tests/src/main/config/common/cookie.properties

```diff
@@ -0,0 +1 @@
+login.token.name=login-token

```

Comment:
I'm just wondering if this needs a separate config file. It's only one prop which is bound to AEM-based projects only. Maybe it would fit in the instances.properties file?